cmd/thanos/bucket.go

```diff
@@ -229,3 +247,31 @@ func runBucketList(gcsBucket, format string) error {
 	}
 	return bkt.Iter(ctx, "", printBlock)
 }
+
+func getBucketClient(gcsBucket *string, s3config s3.Config, reg *prometheus.Registry) (objstore.Bucket, func() error, error) {
+	var (
+		bkt     objstore.Bucket
+		closeFn = func() error { return nil }
+	)
+
+	// initialize object storage
```

Comment:
Please keep comments as valid sentences. (trailing dot and uppercase first letter)

Reply:
Fixed!